### PR TITLE
Implement scenario with rhsso in ha and remote caches on jdg server

### PR DESCRIPTION
**Pull request by guidograzioli**
- keycloak creates a postgresDS now (to allow ha)
- jdg: allow to initialize keycloak caches (hotrod connector, auth but no ssl)
- keycloak: allow to declare remote operation for cache in jdg (via hotrod)
- the demo still has a single keycloak node and a single jdd now, with ha operation enabled

### CODE REVIEW

**Comment by guidograzioli** [comment]
sorry it is kinda big changeset

File: roles/3trains_jdg/defaults/main.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jdg:
   service:
     name: "{{ override_jdg_service_name | default('jdg') }}"
   users:
-    - { name: 'rpelisse', password: 'itsme', roles: 'supervisor' }
+    - { name: 'supervisor', password: 'itsme', roles: 'supervisor' }
```
Comment (rpelisse) [inline roles/3trains_jdg/defaults/main.yml:34]: :D Thanks!

File: roles/3trains_jdg/templates/infinispan.xml.j2 [hunk 2]
```diff
@@ -28,7 +28,9 @@
    <server xmlns="urn:infinispan:server:12.1">
       <interfaces>
          <interface name="public">
-            <inet-address value="${infinispan.bind.address:127.0.0.1}"/>
+             <!--match-interface value="{{ ansible_default_ipv4.interface }}"/-->
+             <!--inet-address value="${infinispan.bind.address:127.0.0.1}"/-->
+             <any-address/>
```
Comment (rpelisse) [inline roles/3trains_jdg/templates/infinispan.xml.j2:31]: Maybe we can remove the two commented out lines?

File: roles/3trains_jdg/defaults/main.yml [hunk 3]
```diff
@@ -11,11 +11,21 @@ jdg:
...
+  user:
     name: "{{ override_jdg_user | default('jdg') }}"
-  group: 
+  group:
     name: "{{ override_jdg_group | default('jdg') }}"
   service:
     name: "{{ override_jdg_service_name | default('jdg') }}"
   users:
-    - { name: 'rpelisse', password: 'itsme', roles: 'supervisor' } 
+    - { name: 'rpelisse', password: 'itsme', roles: 'supervisor' }
```
Comment (rpelisse) [inline roles/3trains_jdg/defaults/main.yml]: @guidograzioli you remove my username elsewhere, shouldn't it also go away here ?

**Comment by rpelisse** [comment]
@guidograzioli No problem! I did a few comment and there is probably things we can enhance later onn but for a first draft, it's good enough! Feel free to merge and test (on the demo env)!

**Pull request merged**